python/pyspark/sql/functions.py

```diff
@@ -10113,6 +10113,120 @@ def unwrap_udt(col: "ColumnOrName") -> Column:
     return _invoke_function("unwrap_udt", _to_java_column(col))
 
 
+@try_remote_functions
```

Comment:
Should also add one in `pyspark.sql.connect.functions` too.

Reply:
Ok - thanks for calling that out, will do!

Reply:
@HyukjinKwon I've added the functions as requested - could you take a quick look and make sure they look good to you?